packages/cli/src/lib/webpack/prerender.js

```diff
@@ -45,112 +45,98 @@ async function handlePrerenderError(err, env, stack, entry) {
-	process.stderr.write(red(`\n${errorMessage}\n`));
+	try {
+		sourceMapContent = JSON.parse(readFileSync(`${entry}.map`, 'utf-8'));
+	} catch (err) {
+		process.stderr.write(red(`\n\nUnable to read sourcemap: ${entry}.map\n`));
+	}
 
-	// If a methodName exists, it's likely user code
-	if (methodName) {
-		let sourceMapContent,
-			position,
-			sourcePath,
-			sourceLines,
-			sourceCodeHighlight;
-		try {
-			sourceMapContent = JSON.parse(readFileSync(`${entry}.map`, 'utf-8'));
-		} catch (err) {
-			process.stderr.write(red(`\n\nUnable to read sourcemap: ${entry}.map\n`));
-		}
+	if (sourceMapContent) {
+		position = await SourceMapConsumer.with(sourceMapContent, null, consumer =>
+			consumer.originalPositionFor({
+				line: stack.getLineNumber(),
+				column: stack.getColumnNumber(),
+			})
+		);
 
-		if (sourceMapContent) {
-			await SourceMapConsumer.with(sourceMapContent, null, consumer => {
-				position = consumer.originalPositionFor({
-					line: stack.getLineNumber(),
-					column: stack.getColumnNumber(),
-				});
-			});
-
-			if (position.source) {
-				position.source = position.source
-					.replace('webpack://', '.')
-					.replace(/^.*~\/((?:@[^/]+\/)?[^/]+)/, (s, name) =>
-						require
-							.resolve(name)
-							.replace(/^(.*?\/node_modules\/(@[^/]+\/)?[^/]+)(\/.*)$/, '$1')
-					);
-
-				sourcePath = resolve(env.src, position.source);
-				sourceLines;
+		if (position.source) {
+			position.source = position.source
+				.replace('webpack://', '.')
+				.replace(/^.*~\/((?:@[^/]+\/)?[^/]+)/, (s, name) =>
+					require
+						.resolve(name)
+						.replace(/^(.*?\/node_modules\/(@[^/]+\/)?[^/]+)(\/.*)$/, '$1')
+				);
+
+			sourcePath = resolve(env.src, position.source);
+			sourceLines;
+			try {
+				sourceLines = readFileSync(sourcePath, 'utf-8').split('\n');
+			} catch (err) {
 				try {
-					sourceLines = readFileSync(sourcePath, 'utf-8').split('\n');
+					sourceLines = readFileSync(
+						require.resolve(position.source),
+						'utf-8'
+					).split('\n');
 				} catch (err) {
-					try {
-						sourceLines = readFileSync(
-							require.resolve(position.source),
-							'utf-8'
-						).split('\n');
-					} catch (err) {
-						process.stderr.write(
-							red(`\n\nUnable to read file: ${sourcePath}\n`)
-						);
-					}
+					process.stderr.write(red(`\n\nUnable to read file: ${sourcePath}\n`));
 				}
-				sourceCodeHighlight = '';
 			}
+			sourceCodeHighlight = '';
+		}
 
-			if (sourceLines) {
-				for (var i = -4; i <= 4; i++) {
-					let color = i === 0 ? red : yellow;
-					let line = position.line + i;
-					let sourceLine = sourceLines[line - 1];
-					sourceCodeHighlight += sourceLine ? `${color(sourceLine)}\n` : '';
-				}
+		if (sourceLines) {
+			for (var i = -4; i <= 4; i++) {
+				let color = i === 0 ? red : yellow;
+				let line = position.line + i;
+				let sourceLine = sourceLines[line - 1];
+				sourceCodeHighlight += sourceLine ? `${color(sourceLine)}\n` : '\n';
```

Comment:
Rather importantly, we weren't handling blank lines correctly as the ternary was falling back to `''`, instead of the new line as it does above. This caused further issues in line positioning.